modules/manage/partials/tiered-storage.adoc

```diff
@@ -575,9 +576,24 @@ Microsoft ABS/ADLS::
 +
 --
 
-
 include::manage:partial$azure-blob-limitations.adoc[]
```

Comment:
@andijcr Please also refer to the deploy preview, but this is a note that says:

> Starting in release 23.2.8, Redpanda supports storage accounts configured for ADLS Gen2 with [hierarchical namespaces](https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-namespace) enabled. For hierarchical namespaces created with a custom endpoint, set cloud_storage_azure_adls_endpoint and cloud_storage_azure_adls_port. If you haven’t configured custom endpoints in Azure, there’s no need to edit these properties.

Just want to confirm that this information is still relevant and doesn't need to be updated.

Reply:
yes, still relevant